### DISCUSSION TITLE
Intel 8080 CPU

### DISCUSSION THREAD

**Discussion by Mukhash2012** [SebLague/Digital-Logic-Sim#218] [Show and tell]
I have just joined in and saw jdaless work (Intel 8008), i wanted to use it and then upgrade it to its successor (Intel 8080).
I am now working on Intel 8085, i am pretty sure that there are some mistakes i made in Intel 8080 and you can help me by leaving a comment.
Data sheet i worked on:
[INTEL 8080A](http://kazojc.com/elementy_czynne/IC/INTEL-8080A.pdf)
[Intel 8080 Microcomputer Systems Users Manual](https://bitsavers.trailing-edge.com/components/intel/MCS80/98-153B_Intel_8080_Microcomputer_Systems_Users_Manual_197509.pdf)

Operation codes (OPCODES):
[8080](https://pastraiser.com/cpu/i8080/i8080_opcodes.html)
[8008 to compare](https://pastraiser.com/cpu/i8008/i8008_opcodes.html)

The CPU (with fast clocks):
[8080.zip](https://github.com/user-attachments/files/18097377/8080.zip)

I hope you will enjoy...
Credits to: jdaless and Sebastian Lague.
![Снимок экрана 2024-12-11 180326](https://github.com/user-attachments/assets/cae297e6-5643-4c4b-9ab4-70444c5db5cc)

**Comment by Mukhash2012** [comment]
Update: i have already made an Intel 8085 but i am checking on some significant issues. It is looks like there are some issues with status signals in Intel 8085
Download:
[8085.zip](https://github.com/user-attachments/files/18100152/8085.zip)

When i am sure that i doesn't have any issues i will post an discussion.

**Reply by Mukhash2012** [reply]
8085 is released: newer zip contains much more bug fixes

**Comment by liam1581** [comment]
I’m to not understanding to do anything with it but I want to haha, could you maybe make some kind of tutorial ish thing where you explain how it works and how I can do stuff with the instructions etc.?

**Reply by Mukhash2012** [reply]
I already created 8088 CPU, i highly recommend to read the 8088 discussion.
Note: every cpu has wrong instruction set, only 8088 CPU has right instruction set, because my main goal was 8088.

**Reply by liam1581** [reply]
ok, thx
